test/spec/fetch-request.js

```diff
@@ -72,6 +72,24 @@ describe('fetchRequest', function () {
         expect(fetchSpy).not.toHaveBeenCalled();
       });
     });
+    it('fetchRequest returns a promise with finally on it', () => {
+      const globalFetch = jest.fn(() => {
+        return Promise.resolve(response);
+      });
+      window.fetch = globalFetch;
+      const fetchRequestPromise = fetchRequest(requestMethod, requestUrl, {});
+      expect(fetchRequestPromise.finally).toBeDefined();
+    });
+    it('fetchRequest returns a promise with finally on it even if fetch doesnt return a promise with fetch', () => {
+      const globalFetch = jest.fn(() => {
+        const resolvedPromise = Promise.resolve(response);
```

Comment:
In this case can we return a bare object? or
```
{
  then: jest.fn(),
  catch: jest.fn()
 }
 ```
 I want to be sure `finally` cannot be called through the prototype, it should never exist.